ntbs-service/Models/Entities/Notification.cs

```diff
@@ -119,17 +119,21 @@ public bool ShouldBeClosed()
                 && mostRecentTreatmentEvent?.TreatmentOutcomeId != null
                 && mostRecentTreatmentEvent.TreatmentEventTypeIsOutcome)
             {
-                if (mostRecentTreatmentEvent.TreatmentOutcome == null)
+                return NotStillOnTreatmentAndOlderThanOneYear(mostRecentTreatmentEvent);
```

Comment:
This can be further simplified by the observation that
```
if (a) { return b; }
return false;
```
where `a` and `b` are booleans, is equivalent to
```
return a && b;
```

Reply:
Good spot, done